Content.Shared/SS220/CultYogg/Buildings/CultYoggBuildingPrototype.cs

```diff
@@ -7,8 +7,7 @@
 
 namespace Content.Shared.SS220.CultYogg.Buildings;
 
-[Prototype("cultYoggBuilding")]
-[Serializable, NetSerializable]
+[Serializable, Prototype]
 public sealed partial class CultYoggBuildingPrototype : IPrototype
```

Comment:
```suggestion
[Prototype]
[Serializable] 
public sealed partial class CultYoggBuildingPrototype : IPrototype
```
Я правда не знаю что требует сириализации но не требует NetSerializable, но я тебе верю что надо